dbfit-java/hsqldb/src/main/java/dbfit/environment/HSQLDBEnvironment.java

```diff
@@ -57,12 +57,9 @@ protected String getConnectionString(String dataSource, String database) {
         throw new UnsupportedOperationException();
     }
 
-    private static final String paramNamePattern = "@([A-Za-z0-9_]+)";
-    private static final Pattern paramRegex = Pattern.compile(paramNamePattern);
-
-    @Override
-    protected Pattern getParameterPattern() {
-        return paramRegex;
+    protected String parseCommandText(String commandText) {
+        commandText = commandText.replaceAll(getParamPatternString(), "?");
+        return super.parseCommandText(commandText);
```

Comment:
@javornikolov I'm thinking that now I've got this block in every adapter I should merge it into the super-class's method.